modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/package.html

```diff
@@ -0,0 +1,8 @@
+<HTML>
+<BODY>
+Integration with Restlet Cloud platform. Help document and manage your web API.
+
+@since Restlet 2.3
```

Comment:
I wonder if we can precise that the extension has been renamed from apispark to platform.

Reply:
Why not. I have absolutely no idea where this is shown in the end though ?